client/src/components/ExploreGames/index.js

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
you need to go into vs code and review the merge conflicts before you do the pull request. just be careful not to overwrite anything from the last push unless it's an update you're making to that file.

Reply:
those are not on my local copy (neither my master or my branch checkout) not sure how they got committed because I did review them.